Mathlib/Data/Set/Intervals/Image.lean

```diff
@@ -15,90 +15,233 @@ variable {f : α → β}
 
 open Set
 
-section
-variable [Preorder α] [Preorder β]
+section Preorder
+variable [Preorder α] [Preorder β] {a b : α}
 
-theorem Monotone.mapsTo_Icc (h : Monotone f) :
-    MapsTo f (Icc a b) (Icc (f a) (f b)) :=
-  fun _ _ => by aesop
+lemma MonotoneOn.mapsTo_Ici (h : MonotoneOn f (Ici a)) : MapsTo f (Ici a) (Ici (f a)) :=
+λ _c hc ↦ by aesop
```

Comment:
Let's try to reduce the diff a bit:
```suggestion
theorem MonotoneOn.mapsTo_Ici (h : MonotoneOn f (Ici a)) : MapsTo f (Ici a) (Ici (f a)) :=
  fun _c hc => by aesop
```
(also `λ` is against the style guide)

Reply:
I meant this globally...

Reply:
I don't think it's going to reduce the diff at all except here.

Reply:
Ah yeah, it doesn't reduce it here because your suggestion should've been
```suggestion
theorem MonotoneOn.mapsTo_Ici (h : MonotoneOn f (Ici a)) : MapsTo f (Ici a) (Ici (f a)) :=
  fun _ _ => by aesop
```

Reply:
Well in fact, the suggestion should be for `Monotone.mapsTo`, which should then make the lemma statements line up.

Reply:
I don't think reducing the diff is a meaningful use of my time: I'm rewriting the entire file, the existing lemmas are getting shuffled (so it's not even like the lemma statements would magically line up) and their proofs are golfed, and I am making sure that the file is consistent style-wise.